<launch>
  <node pkg="four_wheel" type="four_wheel_node" name="four_wheel_node" output="screen">
    <!-- for interoperating with teleop-twist-joy -->
    <remap from="four_wheel_velocity" to="cmd_vel" />
    <!-- for interoperating with dashboard/velocity-calculator -->
    <remap from="four_wheel_odom_info" to="odom" />
    
    <param name="four_wheel_port" type="string" value="/dev/ttyUSB0"/>
    <param name="four_wheel_baudrate" type="int" value="115200"/>

    <param name="four_wheel_odom_enable" type="bool" value="true"/>

    <param name="four_wheel_state_display" type="bool" value="true"/>
    <param name="four_wheel_original_display" type="bool" value="false"/>

    <param name="four_wheel_info_display" type="bool" value="false"/>
    <param name="four_wheel_info_display_time" type="int" value="2"/>

  </node>

  <include file="$(find dashboard)/launch/velocity_calculator.launch">
  </include>

</launch>
